### ISSUE TITLE
SSL for the cloud

### ISSUE THREAD

**Issue by XVilka** [radare/radare2-dockers#3]
for the cloud. What about using LetsEncrypt cert for that?

**Comment by radare** [comment]
yes, the idea was to setup the ssl layer in front of the docker, so, in a
separate machine

On 16 March 2016 at 16:09, Anton Kochkov notifications@github.com wrote:

> for the cloud. What about using LetsEncrypt cert for that?
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> https://github.com/radare/radare2-dockers/issues/3

## 

--pancake

**Comment by r3ad07211** [comment]
openid.net/certification/faq/
